### PR TITLE
Escape the % in the tmux select-pane -t command to prevent process substitution by fish

### PR DESCRIPTION
**Pull request by scolby33**
Re #148

### CODE REVIEW

File: plugin/tmux_navigator.vim [hunk 1]
```diff
@@ -78,7 +78,7 @@ function! s:TmuxAwareNavigate(direction)
       catch /^Vim\%((\a\+)\)\=:E141/ " catches the no file name error 
       endtry
     endif
-    let args = 'select-pane -t ' . $TMUX_PANE . ' -' . tr(a:direction, 'phjkl', 'lLDUR')
+    let args = 'select-pane -t \' . $TMUX_PANE . ' -' . tr(a:direction, 'phjkl', 'lLDUR')
```
Comment (blueyed) [inline plugin/tmux_navigator.vim]: What about `let args = 'select-pane -t ' . shellescape($TMUX_PANE) . ' -' . tr(a:direction, 'phjkl', 'lLDUR')` instead?

Comment (blueyed) [inline plugin/tmux_navigator.vim]: i.e. use `shellescape()` instead of the backslash.

Comment (scolby33) [inline plugin/tmux_navigator.vim]: Good call. It seems that enclosing in quotes works as well as escaping with `\`.

**Comment by scolby33** [comment]
Let me know if you want me to squash commits before merging.

**Comment by blueyed** [comment]
:+1: 

I prefer it squashes, yes.

But we can do so from the GitHub UI.  Feel free to force-push what you like, of course.

Please ping me - giving also time for @christoomey to chime in, but I think this is good to fly.

**Comment by blueyed** [comment]
:fireworks:

**Pull request merged**